cdk-v2/.cdk

```diff
@@ -1,2 +1,3 @@
 openshift.auth.scheme=Basic
 openshift.auth.username=test-admin
+cdk.version=2.0.0
```

Comment:
That's not really the version we are using, right? We are using some temporary build based on 2.0.0 Beta3.